AutomatedTesting/src/test/resources/features/AddPlanetCharacterLimit.feature

```diff
@@ -0,0 +1,12 @@
+Scenario Outline: Add Planet  30 character Limit Requirement Test Case
```

Comment:
Feature Files start with "Feature: ..." on the first line and "Scenario Outline: ..." on the following line indented.